integration-tests/realm-apps/with-db/services/mongodb/config.json

```diff
@@ -8,7 +8,7 @@
       "partition": {
         "key": "partition",
         "type": "string",
-        "required": true,
+        "required": false,
```

Comment:
I see your point here. The other two suggestions I have then is to either add partition values to the objects we create in this test (which I believe only incurs a lot of noise since the partition value is not tested). Or I can create a new app to use with sync and an optional partition key, but I feel like that would also bloat the codebase with another app.. any other suggestions? 🤔